libraries/standard/http/src/http_client.c

```diff
@@ -839,7 +841,9 @@ static int httpParserOnBodyCallback( http_parser * pHttpParser,
     /* coverity[misra_c_2012_rule_18_3_violation] */
     if( pLoc > pNextWriteLoc )
     {
-        ( void ) memcpy( pNextWriteLoc, pLoc, length );
+        /* memmove is used instead of memcpy because memcpy cannot handle
```

Comment:
/* memmove is used instead of memcpy because memcpy has undefined behavior
         * when the src and dest overlap. */